libtsuba/include/tsuba/RDG.h

```diff
@@ -181,15 +181,18 @@ class KATANA_EXPORT RDG {
 
   const FileView& topology_file_storage() const;
 
+  void set_view_name(const std::string& v) { view_type_ = v; }
+
```

Comment:
Do we want views to be strings?  Or class enums convertible to strings?

Reply:
strings until structure emerges imo

Reply:
Slight preference for enums here. It mostly serves a documentation purpose, I can look at the `enum` and know all of the view types this version of the code supports.

It also stops future refactor-ers from having to ask the question "is the view name guaranteed to be valid here".

I don't know what structure would emerge other than 'this is a list of the view types we support', but if you feel strongly I won't block this over that.

Reply:
Right now we have the partitioned RDG view type. In my current implementation I treat the variations in partitioned RDGs as distinct types. That means type(4-way partitioned RDG) != type(8-way partitioned RDG). Simplifies the query "is there this view available?" to constructing a string and scanning for filenames which match this string. 

I also dont mind refactoring to have the enums and view type construction if you strongly prefer. My preference is to get it in first and then make these tweaks after steady-state emerges.